src/sentry_plugins/splunk/plugin.py

```diff
@@ -303,13 +243,9 @@ def forward_event(self, event, payload):
                 },
             )
 
-            if isinstance(exc, (ConnectTimeout, ReadTimeout)):
-                # If we get a ConnectTimeout or ReadTimeout we don't need to raise an error here.
+            if isinstance(exc, (ApiHostError, ApiTimeoutError, ApiError)):
```

Comment:
@NisanthanNanthakumar For `ApiError`, shouldn't we check the status code being `403` like we had before?

Reply:
@scefali true, missed that!